test/ci/xyk-ci.validate.assetInfo.test.ts

```diff
@@ -49,3 +51,18 @@ test("xyk-CI - AssetInfo contains assets for MGA, mKSM, mBTC and mUSD", async ()
     assetsInfo.findIndex((asset) => asset.symbol === "mUSD")
   ).toBeGreaterThanOrEqual(0);
 });
+
+test("xyk-CI - Check pools for MGA-mKSM, MGA-mBTC and MGA-mUSD", async () => {
+  const balanceMGAKSM = await getBalanceOfPool(new BN(0), new BN(4));
```

Comment:
I think we have constants for MGA_ASSET_ID and KSM. 
Maybe you can leave others, but plz use at least reuse the ones we already have